contracts/.solhint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "solhint:default",
```

Comment:
There's already a solhint.json at the root of the project. https://github.com/river-build/river/blob/main/.solhint.json

Reply:
Have to make some changes to pass CI.